src/lib/components/Codelab/index.js

```diff
@@ -45,8 +45,8 @@ class Codelab extends BaseElement {
   }
 
   get src() {
-    if (!this.glitch) {
-      return;
+    if (!this.glitch || window.matchMedia('(max-width: 865px)').matches) {
```

Comment:
I appreciate this but I'd rather have a listener that maintains/updates a property on this element, so we can add or remove the codelab if the user changes the window size.

Reply:
@samthor WDYT of https://github.com/GoogleChrome/web.dev/pull/2389/commits/e0c08fae0a23ddeb0e84cca66aa68faf1c97c4eb?

Reply:
@samthor New commit at https://github.com/GoogleChrome/web.dev/pull/2389/commits/84e05226ccde4dc366b3add4e065bc368713b6b6